### PR TITLE
Add Ethereum Foundation YouTube to Footer

### PR DESCRIPTION
**Pull request by edsonayllon**
## Description

Adds link to the official Ethereum YouTube to the footer.

## Related Issue

#664

## Screenshots (if appropriate):

Screen Shot 2019-12-26 at 12 52 37 PM

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8291,10 +8291,10 @@ serialize-javascript@^1.3.0, serialize-javascript@^1.7.0:
   resolved "https://registry.yarnpkg.com/serialize-javascript/-/serialize-javascript-1.9.1.tgz#cfc200aef77b600c47da9bb8149c943e798c2fdb"
   integrity sha512-0Vb/54WJ6k5v8sSWN09S0ora+Hnr+cX40r9F170nT+mSkaxltoE/7R3OrIdBSUv1OoiobH1QoWQbCnAO+e8J1A==
 
-serialize-javascript@^2.1.1:
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/serialize-javascript/-/serialize-javascript-2.1.0.tgz#9310276819efd0eb128258bb341957f6eb2fc570"
-  integrity sha512-a/mxFfU00QT88umAJQsNWOnUKckhNCqOl028N48e7wFmo2/EHpTo9Wso+iJJCMrQnmFvcjto5RJdAHEvVhcyUQ==
+serialize-javascript@^2.1.0:
```
Comment (samajammin) [inline yarn.lock]: Why this change?

Comment (edsonayllon) [reply yarn.lock]: I just ran yarn, yarn dev to see the changes locally, must have updated the dependencies. I can revert those changes later today if you'd like.

Comment (edsonayllon) [reply yarn.lock]: Actually, had time before my flight. 146ee80

**Comment by samajammin** [comment]
Thanks again for this PR & the descriptive issue ❤️

**Comment by samajammin** [comment]
@all-contributors please add @edsonayllon for content
